### PR TITLE
Fix trigger build order

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: script/build [hunk 1]
```diff
@@ -9,6 +9,7 @@ cd "$(dirname "$0")/.."
 mkdir -p ./site/public
 
 if [ -n "$CF_KV_TOKEN" ]; then
+  sleep 70 # Wait for KV writes to reach all nodes
```
Comment (balloob) [inline script/build:12]: check the env variable if we got triggered or if it's based on a PR and only sleep on trigger

Comment (balloob) [reply script/build:12]: Also 61 should be enough.

File: worker/src/handlers/schedule.ts [hunk 2]
```diff
@@ -73,15 +73,15 @@ async function processQueue(): Promise<void> {
...
     core_analytics[timestampString] = queue_data;
 
+    // Trigger Netlify build
+    await fetch(NETLIFY_BUILD_HOOK, { method: "POST" });
+
     await KV.put(
       `${KV_PREFIX_HISTORY}:${timestampString}`,
       JSON.stringify(queue_data)
     );
     await KV.put(KV_KEY_CORE_ANALYTICS, JSON.stringify(core_analytics));
```
Comment (balloob) [inline worker/src/handlers/schedule.ts:83]: I would do this write still before the trigger. And I would also check the return value of the netlify trigger and raise if `resp.ok` is false.

Comment (ludeeus) [reply worker/src/handlers/schedule.ts:83]: We have to do everything or nothing.

**Review by balloob — APPROVED** [review]

**Pull request merged**